### DISCUSSION TITLE
Is it dangerous to start parallel migrations?

### DISCUSSION THREAD

**Discussion by cawa-93** [prisma/prisma#13856] [Q&A] [answered]
I have a web application that deploys to several separate applications in heroku. **Each such application is connected to a single database**. They are built from source code and **trigger migration**. Theoretically, it is possible that two such applications will be built at the same time and start migrations at the same time. I'm worried whether this will cause irreparable damage to the database?

**Comment by nurul3101** [comment, marked as answer]
Hey @cawa-93 👋 

Prisma Migrate makes use of advisory locking when you run production commands, This safeguard ensures that multiple commands cannot run at the same time - for example, if you merge two pull requests in quick succession.

You can learn more about Advisory Locking in this reference: [Advisory Locking](https://www.prisma.io/docs/concepts/components/prisma-migrate#advisory-locking)

**Reply by cawa-93** [reply]
Sounds fantastic

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
